File: kits/kotlin/simple/bot/src/main/kotlin/luxaibot/lux/Cell.kt
package luxaibot.lux

class Cell(x: Int, y: Int) {
    var pos: Position
    var resource: Resource? = null
    var road = 0.0
    var citytile: CityTile? = null
    fun hasResource(): Boolean {
        return resource != null && resource!!.amount > 0
    }

    init {
        pos = Position(x, y)
    }
}

File: kits/csharp/simple/Bot/Lux/Cell.cs
namespace Bot.Lux
{
	public class Cell
	{
		public Position Pos;
		public Resource Resource = null;
		public double Road = 0;
		public CityTile CityTile = null;

		public Cell(int x, int y)
		{
			Pos = new Position(x, y);
		}
		public bool HasResource()
		{
			return Resource != null && Resource.Amount > 0;
		}
	}
}
